src/containers/ConfirmPassword/index.js

```diff
@@ -1,3 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'recompose';
+import { Form } from 'antd';
+import { withAuth } from '../Session/index';
+
+import { withFirebase } from '../Firebase';
+import { EDIT_ACCOUNT } from '../../constants/routes';
+
 import ConfirmPassword from './confirmPass';
 
-export default ConfirmPassword;
+class Index extends Component {
+  state = {
+    errorMessage: '',
+    loading: false,
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const {
+      form: { validateFields },
+      firebase,
+      history: { push },
+    } = this.props;
+    validateFields(async (err, values) => {
+      if (!err) {
+        this.setState({ loading: true });
+        try {
+          const user = await firebase.auth.currentUser;
+          await firebase.doSignInWithEmailAndPassword(
+            user.email,
+            values.password
+          );
+          localStorage.setItem('confirm', true);
+          this.setState({ loading: false });
+          push(EDIT_ACCOUNT);
+        } catch (error) {
+          if (error) this.setState({ errorMessage: error.message });
```

Comment:
here also you should setState to the loading to be false
```suggestion
          if (error) this.setState({ errorMessage: error.message, loading: false  });
```